common/journal_entries/imperia_carlist_war_je.txt

```diff
@@ -0,0 +1,69 @@
+﻿je_carlist_war = {
+	icon = "gfx/interface/icons/event_icons/waving_flag.dds"
+
+	group = je_group_foreign_affairs
+
+	should_be_pinned_by_default = yes
+
+	modifiers_while_active = {
+		modifier_carlist_war_funding
+	}
+
+	complete = {
+		is_at_war = no
+		owns_entire_state_region = STATE_TOLEDO
+	}
+	
+	on_complete = {
+		change_tag = SPA
+		c:SPA = {
+			create_diplomatic_pact = {
+				country = c:PHI
+				type = puppet
+			}
+		}
+		c:SPA = {
+			create_diplomatic_pact = {
+				country = c:CUB
+				type = puppet
+			}
+		}
```

Comment:
Is this manual subject transfer necessary (as the wargoal should already do it)? And if it is, can't you write a script that goes through all subjects and takes them over instead of hardcoding them?